src/App.vue

```diff
@@ -4,11 +4,11 @@
     class="app-container"
   >
     <!-- Dialogs -->
-    <NameRequestInvalidErrorDialog
+    <InvalidIncorpAppDialog
```

Comment:
I renamed this dialog because the name didn't match the dialog title/function.

(And also, I wanted to use a similar name for a new error dialog.)